tests/loggers/test_wandb_logger.py

```diff
@@ -40,6 +40,8 @@ def download(self, root: pathlib.Path):
 def test_wandb_logger(tmp_path, dummy_state):
     pytest.importorskip('wandb', reason='wandb is optional')
     os.environ['WANDB_DIR'] = str(tmp_path)
+    if not tmp_path.exists():
+        tmp_path.mkdir()
```

Comment:
Why does this need to happen? Shouldn't it always exist?

Reply:
ya I can  remove this, this solution was from our manual deletion of tmp_dir solution